### PR TITLE
Taboo: Updated GAME-STATUS and GAME-OVER messages

### PR DESCRIPTION
**Pull request by lambdaFooBar**
GAME-STATUS message includes info on connected players and the winners if applicable
GAME-OVER message includes winner list. TODO winner list not populated yet.

### CODE REVIEW

File: src/Taboo/TabooPlayer.py [hunk 1]
```diff
@@ -38,3 +38,7 @@ def ready(self):
     @ready.setter
     def ready(self, flag):
         self.__ready = flag
+
+    @property
+    def nrofConns(self):
```
Comment (sbirmi) [inline src/Taboo/TabooPlayer.py]: nit: numConns() perhaps?

File: src/Taboo/TabooPlayer.py [hunk 2]
```diff
@@ -38,3 +38,7 @@ def ready(self):
     @ready.setter
     def ready(self, flag):
         self.__ready = flag
+
+    @property
```
Comment (sbirmi) [inline src/Taboo/TabooPlayer.py]: suggestion: remove property?

The rationale might seem arbitrary but I typically use @property as accessors for "self._attr". If we are computing state at run-time, perhaps it's better to make it visible as a function call.

functional-call also makes it apparent that the attribute is not mutable. It's not clear immediately from the users point of view if obj.nrofConns is mutable just by looking at it; "obj.nrofConns()" is certainly not mutable!)

**Review by sbirmi — COMMENTED** [review]
Changes look good. Please feel free to merge this!

File: src/Taboo/TabooPlayer.py (hunk 1 above)
Comment (lambdaFooBar) [reply src/Taboo/TabooPlayer.py]: Ok

File: src/Taboo/TabooPlayer.py (hunk 2 above)
Comment (lambdaFooBar) [reply src/Taboo/TabooPlayer.py]: I agree.

**Pull request merged**